components/camera/fake/camera.go

```diff
@@ -346,33 +343,25 @@ func (c *Camera) Unsubscribe(ctx context.Context, id rtppassthrough.Subscription
 }
 
 func (c *Camera) startPassthrough() error {
-	forma := &format.H264{
-		PayloadTyp:        96,
-		PacketizationMode: 1,
-		SPS: []uint8{
-			0x67, 0x64, 0x0, 0x15, 0xac, 0xb2, 0x3, 0xc1, 0x1f, 0xd6,
-			0x2, 0xdc, 0x8, 0x8, 0x16, 0x94, 0x0, 0x0, 0x3, 0x0, 0x4, 0x0, 0x0, 0x3, 0x0,
-			0xf0, 0x3c, 0x58, 0xb9, 0x20,
-		},
-		PPS: []uint8{0x68, 0xeb, 0xc3, 0xcb, 0x22, 0xc0},
+	forma := &format.H264{}
+	webrtcPayloadMaxSize := 1188 // 1200 - 12 (RTP header)
+	encoder := &rtph264.Encoder{
+		PayloadType:    96,
+		PayloadMaxSize: webrtcPayloadMaxSize,
```

Comment:
This change is needed to prevent RTP packets from being dropped due to being over the webrtc packet size limit of 1200 bytes.